src/pages/_assets/welcome-content.tsx

```diff
@@ -0,0 +1,267 @@
+      language: "verify",
+      languageLogoAlt: "Verification icon",
+      docs: "/docs/kratos/self-service/flows/verify-email-account-activation",
+    },
+    {
+      title: "Manage user sessions",
+      description: "Control login sessions, timeouts, and multi-device access",
+      language: "session",
+      languageLogoAlt: "Session icon",
+      docs: "/docs/kratos/session-management/overview",
+    },
+    {
+      title: "Support multi-factor authentication",
+      description: "Add TOTP, WebAuthn, or SMS verification",
+      language: "shield",
+      languageLogoAlt: "MFA icon",
+      docs: "/docs/kratos/mfa/overview",
+    },
+  ],
+}
+
+// Section 2: Authorization & Security
+export const authorizationSecurity: {
+  id: string
+  title: string
+  description: string
+  cards: CardItem[]
+} = {
+  id: "authorization-security-section",
+  title: "Authorization & security",
+  description: "How to secure access to resources and APIs",
+  cards: [
+    {
+      title: "Implement OAuth flows",
+      description: "Add authorization code, client credentials, or other flows",
+      language: "oauth",
+      languageLogoAlt: "OAuth icon",
+      docs: "/docs/oauth2-oidc/authorization-code-flow",
+    },
+    {
+      title: "Enable organization login",
+      description:
+        "Group users by organization and support corporate SSO with SAML and OIDC providers",
+      language: "organization",
+      languageLogoAlt: "Organization icon",
+      docs: "/docs/kratos/organizations",
+    },
+    {
+      title: "Manage access tokens",
+      description: "Issue, validate and revoke access credentials",
+      language: "token",
+      languageLogoAlt: "Token icon",
+      docs: "/docs/guides/token-management",
+    },
+    {
+      title: "Secure APIs",
+      description:
+        "Protect your backend services with token-based authentication",
+      language: "api",
+      languageLogoAlt: "API icon",
+      docs: "/docs/getting-started/ory-network-oauth2#authorization-code-grant",
+    },
+    {
+      title: "Connect service accounts",
+      description: "Enable machine-to-machine authentication",
+      language: "service",
+      languageLogoAlt: "Service account icon",
+      docs: "/docs/oauth2-oidc/client-credentials",
+    },
+  ],
+}
+
+// Section 3: Permission Management
+export const permissionManagement: {
+  id: string
+  title: string
+  description: string
+  cards: CardItem[]
+} = {
+  id: "permission-management-section",
+  title: "Permission management",
+  description: "Controlling who can access what",
+  cards: [
+    {
+      title: "Check access permissions",
+      description: "Verify if users can perform specific actions",
+      language: "permission",
+      languageLogoAlt: "Permission icon",
+      docs: "/docs/keto/guides/simple-access-check-guide",
+    },
+    {
+      title: "Implement role-based access",
+      description: "Control access based on user roles and groups",
```

Comment:
I am not sure if this guide is actually working still - can you maybe test it to make sure? Last time I tried it I found it too outdated to be useful. 
IMO it you can't get it to work without issues we should not include it on the frontpage until its updated.